Sources/SwiftInspectorVisitors/ExtensionVisitor.swift

```diff
@@ -66,55 +66,15 @@ public final class ExtensionVisitor: SyntaxVisitor {
   }
 
   public override func visit(_ node: ClassDeclSyntax) -> SyntaxVisitorContinueKind {
-    if !hasFinishedParsingExtension, let extensionInfo = extensionInfo {
```

Comment:
These three method bodies looked _exactly_ the same. And now that we have a single protocol representing a `NestableDeclSyntax`, we could write this code once and reuse it.